### PR TITLE
[chores] Updated README and standardized URL #80

### LINKED ISSUE #80: [docs] Update README
Update setup instructions in README. 
Explicitly mention ordering of INSTALLED_APPS.

### PR DESCRIPTION
**Pull request by pandafy**
Updated setup instructions
Standardized API URLS
Closes #80

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -664,21 +674,21 @@ Get notification details
 
 .. code-block:: text
 
-    GET /api/v1/notifications/{pk}/
+    GET /api/v1/notifications/notification/{pk}/
```
Comment (nemesisdesign) [inline README.rst:677]: `/api/v1/notifications/` should become `/api/v1/notification/`

Comment (pandafy) [reply README.rst:677]: But according to following, the convention seems to be
```
/api/v1/<app-name>/<model-name>/{id}
```
image

Comment (nemesisdesign) [reply README.rst:677]: the main point you should notice is that all the URLs use the singular noun of the model, not the plural

**Comment by pandafy** [comment]
Screenshot_2020-08-11 OpenWISP API(1)


It will look weirder now in my opinion. :sweat_smile:

**Comment by nemesisdesign** [comment]
> Screenshot_2020-08-11 OpenWISP API(1)
> 
> It will look weirder now in my opinion.

it will look less weird when you see it with the rest of the openwisp modules.

There's still other inconsistencies to resolve but at least this one we resolved it at the root.

**Pull request merged**